docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -9,9 +9,7 @@ packageName: '@mui/x-date-pickers'
 
 <p class="description">This page explains when the onChange, onAccept, and onClose callbacks are called.</p>
 
-## Fields lifecycle
-
-### On simple fields
+## Lifecycle on simple fields
```

Comment:
I think this is the first place where we have used the notion of `simple fields`.
WDYT about going with just `fields` to reduce the uncertainty? 🤔

Reply:
We refer to the non range as "single item value" in some places

I'm fine with "fields", it's not strictly speaking correct since it's not all the fields, but I think it's clear enough